webpack/ForemanTasks/Components/TasksTable/TasksTable.js

```diff
@@ -50,11 +40,31 @@ const TasksTable = ({
     return <span>{__('No Tasks')}</span>;
   }
 
+  const changeTablePage = ({ page, perPage }) => {
+    updateURlQuery(
+      {
+        page,
+        per_page: perPage,
+      },
+      history
+    );
+  };
+
+  const setSortHistory = (by, order) => {
+    updateURlQuery({ sort_by: by, sort_order: order }, history);
+  };
+
+  const uriQuery = getURIQuery(window.location.href);
```

Comment:
Can't it use the `history` object as well?

Reply:
You're right! changed here and in other places that I missed as well